src/main/webapp/help/Trigger/help-pathsToWatch.html

```diff
@@ -16,4 +16,5 @@
 `
 <div>
```

Comment:
As @Kuanlin-Chen suggested, let'e remove the dot above this line.

Reply:
Done